app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    splitted = command.split(" ")
```

Comment:
The task checklist specifies that you should ensure the input string contains exactly three elements separated by spaces and that the first element is 'cp'. Consider adding a check to validate the input format before proceeding with the rest of the function logic.